src/ReverseProxy/Management/ReverseProxyServiceCollectionExtensions.cs

```diff
@@ -30,6 +30,7 @@ public static IServiceCollection AddHttpForwarder(this IServiceCollection servic
     {
         services.TryAddSingleton<IClock, Clock>();
         services.TryAddSingleton<IHttpForwarder, HttpForwarder>();
+        services.TryAddSingleton<DirectForwardingHttpClientProvider>();
```

Comment:
Since `DirectForwardingHttpClientProvider` is internal and have no interface, there's no intent to replace this service with another service. Perhaps we could `AddSingleton` is enough in this situation.
```suggestion
        services.AddSingleton<DirectForwardingHttpClientProvider>();
```